study.md

```diff
@@ -43,21 +43,21 @@ what do developers mean when they say "view-state"? How do front-end "routers"
 help manage view-state?
 
 ```md
-<!-- your answer here -->
+<!-- View is how many times anyone visits a website.  view state controls how many users can visit a website in a given period of time. -->
 ```
 
 ## Define the advantages of an SPA over a traditional application
 
 What are the advantages of a single page application over a traditional appliction?
 
 ```md
-<!-- your answer here -->
+<!-- less work and mess are required for a single page as to multiple pages -->
```

Comment:
Yes, it makes the user's experience more fluid and responsive as the page doesn't have to fully refresh every time some new data is loaded or the view is changed.